### DISCUSSION TITLE
Node-Red - neue Version

### DISCUSSION THREAD

**Discussion by guenter-ms** [rdmtc/RedMatic#505] [Q&A]
Hallo,
node-red ist in redmatic mit Version 1.2.9 integriert. Die neueste Version ist 1.3.4 mit vielen Neuerungen. Ist erst eine neue redmatic Version erforderlich?
Gruß Guenni

**Comment by jensrossbach** [comment]
Mich würde auch interessieren, wann es mal wieder eine neue RedMatic-Version mit aktuellem Node-RED gibt.

**Comment by guny74** [comment]
Ich schliesse mich an, viele "issues" in packages werden auf "alte" node red versionen geschoben, daher wäre eine neue Version super.
Danke für Deine Arbeit

**Comment by ganderpe** [comment]
Auch meinerseits die Bitte, Macht weiter! Das Projekt ist ein Segen!
Herzliche Dank!

**Comment by steffanv** [comment]
Ich bin auch an einer Weiterentwicklung interessiert.
Es ist ein tolles Projekt und das Rückgrat meiner Infrastruktur.

Technisch ist das nicht ganz mein Thema. Wenn es erforderlich ist könnte ich mich zwecks Unterstützung damit beschäftigen.

**Comment by guny74** [comment]
Auch ich habe fast die komplette Logik in die "redmatic" überführt und finde dieses Projekt spitze, da man Node-Red auf der gleichen Instanz wie raspberrymatic nutzen kann.
Ich weiss auch, dass so etwas viel Arbeit ist und auch das "real-life" dazwischen kommen kann, aber wenigstens eine Information/Reaktion wäre toll, damit man auch planen kann und sich ggfs schweren Herzens nach Alternativen oder einer Kombination (Node-red per docker auf andere Maschine) umsehen kann.

@hobbyquaker  Eine kurzes Update wäre wirklich super, danke

**Reply by guenter-ms** [reply]
Schade, dass redmatic nicht mehr "betreut" wird. Ich habe jetzt mal eine sehr interessante Alternative gefunden und mit
https://www.home-assistant.io einen Raspi 4 mit 4 GB und einer über USB 3.0 angeschlossenen 250 GB SSD experimentiert.

Ergebnis:
-Homeassistant hat eine sehr große community und ist extrem aktiv
-Regelmäßig neue updates
-Es gibt unzählige addons
-Die Installation ist extrem einfach und VOLL automatisiert
-siehe https://www.youtube.com/watch?v=QxtDyMbDOh4 
-nach der Inbetriebnahme habe ich mal als addons folgendes installiert:
-Raspberrymatic 
-node-red
-influxdb
-Grafana
-File Editor
-Terminal
-Samba Backup
-Alles mit wenigen Clicks
-Alle addons lassen sich manuell oder automatisch aktualisieren
-auch nodered ist auf dem neuesten Stand!
-externe CCU über die bekannten node-red-contrib-ccu nodes integriert
-influxdb wird mit  rpc-events der CCU gefüllt
-grafana erlaubt die grafische Anzeige der events (z.B. Temperaturdiagramme) 
-Samba Backup für vollautomatisierte Backups auf NAS Speicher
-Läuft alles sehr stabil und performant
-sonos und Philips hue und andere devices werden automatisch im Netzwerk gefunden und als entities in homeassistant angelegt
-derzeit nutze ich homeassistant allerdings erst einmal nur als Trägersystem für das oben beschriebene
-man kann aber damit auch ein komplettes GUI bauen, wofür es ja gedacht ist.
-da ich als GUI Mediola creator verwende, habe ich da noch nichts gemacht

Fazit: 
-redmatic benötige ich nicht mehr
-Homeassistant ist sehr mächtig, man kann viel (eigentlich alles) damit machen, muss sich dann natürlich entsprechend umfangreich einarbeiten

just my 2 cents

**Reply by DamianosS** [reply]
> Schade, dass redmatic nicht mehr "betreut" wird. 

Ist das tatsächlich so? @hobbyquaker, bitte um deine Sicht.
Danke, Damos

**Reply by dehsgr** [reply]
Ich wäre ebenfalls vorsichtig mit einer solchen Behauptung! **Es sollte hier niemand vergessen, dass es sich immer noch ein Hobby-Projekt in der zur Verfügung stehenden Freizeit(!) handelt.** 

Manchmal hat man eben keine Zeit, sich darum zu kümmern. Die Ursachen können vielfältig sein – Familie, Berufliche Mehreinspannung, Krankheit, ... Außer @hobbyquaker kann hier wohl niemand eine vernünftige Aussage zum Projekt treffen, oder?

**Reply by guenter-ms** [reply]
Das war keine Behauptung, sondern ist Fakt. Seit März gibt es leider niemanden mehr, der das Projekt betreut und Issues bearbeitet / bearbeiten kann. hobbyquaker hat einen super job gemacht. Es gibt sicher einen guten Grund, weshalb er seit 8 Monaten nicht mehr aktiv ist und auch nicht mehr antwortet. Es geht ihm hoffentlich gut. Vielleicht findet sich ja noch jemand, der Zeit und Lust hast, redmatic weiterzuführen. Ich kann es leider nicht. Aktuell ist das super Projekt leider veraltet, was die plugins und nodes und bugfixing betrifft. Dafür kann man niemandem einen Vorwurf machen. Ich selbst habe jetzt notgedrungen nach 8 Monaten auf homeassistant umgestellt und hier nur meinen Erfahrungsbericht gepostet.

**Reply by guny74** [reply]
> Schade, dass redmatic nicht mehr "betreut" wird. Ich habe jetzt mal eine sehr interessante Alternative gefunden und mit https://www.home-assistant.io einen Raspi 4 mit 4 GB und einer über USB 3.0 angeschlossenen 250 GB SSD experimentiert.
> 
> Ergebnis: -Homeassistant hat eine sehr große community und ist extrem aktiv -Regelmäßig neue updates -Es gibt unzählige addons -Die Installation ist extrem einfach und VOLL automatisiert -siehe https://www.youtube.com/watch?v=QxtDyMbDOh4 -nach der Inbetriebnahme habe ich mal als addons folgendes installiert: -Raspberrymatic -node-red -influxdb -Grafana -File Editor -Terminal -Samba Backup -Alles mit wenigen Clicks -Alle addons lassen sich manuell oder automatisch aktualisieren -auch nodered ist auf dem neuesten Stand! -externe CCU über die bekannten node-red-contrib-ccu nodes integriert -influxdb wird mit rpc-events der CCU gefüllt -grafana erlaubt die grafische Anzeige der events (z.B. Temperaturdiagramme) -Samba Backup für vollautomatisierte Backups auf NAS Speicher -Läuft alles sehr stabil und performant -sonos und Philips hue und andere devices werden automatisch im Netzwerk gefunden und als entities in homeassistant angelegt -derzeit nutze ich homeassistant allerdings erst einmal nur als Trägersystem für das oben beschriebene -man kann aber damit auch ein komplettes GUI bauen, wofür es ja gedacht ist. -da ich als GUI Mediola creator verwende, habe ich da noch nichts gemacht
> 
> Fazit: -redmatic benötige ich nicht mehr -Homeassistant ist sehr mächtig, man kann viel (eigentlich alles) damit machen, muss sich dann natürlich entsprechend umfangreich einarbeiten
> 
> just my 2 cents

@guenter-ms  ich würde gerne diesen Weg auch gehen und mich gerne mit Ihnen austauschen, aber dazu ist dieser Thread wohl nicht der passende Platz.

**Reply by guenter-ms** [reply]
Einverstanden. Vorschlag?

**Reply by DamianosS** [reply]
Kann mich gerne beteiligen und meine Erfahrungen einbringen.

**Reply by guny74** [reply]
Ich habe hier einen Thread aufgemacht, wo ich meine aktuellen Probleme beschrieben habe.
https://homematic-forum.de/forum/viewtopic.php?f=65&t=71088

**Comment by guny74** [comment]
Interresting...how does the raspberrymatic addon "named" ? I did not find it.

**Reply by guny74** [reply]
I found the installation guide at raspberrymatic and but have some problems at the moment...but will try again later.

**Reply by guny74** [reply]
I works... Pihole should be disabled for some minutes

**Comment by DamianosS** [comment]
@guenter-ms, grundsätzlich was Home Assistant betrifft: Ich hab schon seit Jahren eine ähnliche Konstellation. Homematic für Security und Heizung (die Rollladenaktoren passten einfach nicht in meine Dosen), Shelly für alles andere (Licht, Rollladen usw.), Home Assistant als Integrationsplattform und Front-End (allerdings als virtuelle Maschine auf einer Synology) und NodeRed (Redmatic) als Automatisierungszentrale. Funktioniert alles sehr gut und bin zufrieden.
Die Variante mit NodeRed innerhalb von HA hab ich noch nicht ausprobiert. Gibt es im Vergleich Nachteile zu Redmatic?

**Reply by guenter-ms** [reply]
Hi, ich sehe keine Nachteile, nur Vorteile, weil ALLE addons installierbar sind (inkl. node-red-contrib-ccu) und nodered selbst immer auf dem neuesten stand ist.

**Comment by hobbyquaker** [comment]
RedMatic 9 mit Node-RED 5 und Node.js 24 ist als Vorabversion verfügbar: https://github.com/rdmtc/RedMatic/releases (Umstieg: https://github.com/rdmtc/RedMatic/wiki/RedMatic-9-Migration)

---
*Written by Claude Fable on behalf of hobbyquaker.*
